src/main/kotlin/no/nav/syfo/aktivitetskrav/database/AktivitetskravVarselRepository.kt

```diff
@@ -81,39 +82,49 @@ class AktivitetskravVarselRepository(private val database: DatabaseInterface) {
     ): Int =
         withContext(Dispatchers.IO) {
             database.connection.use { connection ->
-                val rowsAffected = connection.prepareStatement(Queries.setExpiredVarselPublishedAt).use {
-                    it.setObject(1, nowUTC())
-                    it.setObject(2, nowUTC())
-                    it.setString(3, publishedExpiredVarsel.varselUuid.toString())
-                    it.executeUpdate()
-                }
+                val rowsAffected =
+                    connection.prepareStatement(SET_EXPIRED_VARSEL_PUBLISHED_AT).use {
+                        it.setObject(1, nowUTC())
+                        it.setObject(2, nowUTC())
+                        it.setString(3, publishedExpiredVarsel.varselUuid.toString())
+                        it.executeUpdate()
+                    }
                 if (rowsAffected != 1) {
                     throw SQLException("Expected one row to be updated, got update count $rowsAffected")
                 }
                 connection.commit()
                 rowsAffected
             }
         }
-}
 
-private object Queries {
-    const val selectExpiredVarsler =
-        """
-            SELECT a.personident, a.uuid as aktivitetskrav_uuid, varsel.*
+    companion object {
+        private const val AKTIVITETSKRAV_WITH_HANDLED_VARSEL =
+            """
+            SELECT a.id
+            FROM aktivitetskrav_vurdering vurdering
+            WHERE vurdering.aktivitetskrav_id = a.id
+                AND vurdering.created_at > varsel.created_at
+                AND varsel.created_at + (INTERVAL '3 weeks') > vurdering.created_at
```

Comment:
Trengs denne sjekken?

Reply:
Spørs litt hvordan vi ønsker å definere det når vi "lager et nytt aktivitetskrav"/"ny aktivitetskrav vurdering". Hvis vi sjekker om det på noe som helst tidspunkt har kommet en vurdering på `IKKE_AKTUELL`, `OPPFYLT` eller `UNNTAK` så kan det jo hende at man får opp noen aktivitetskrav der det er 1 måned etter varselet ble sendt ut. Da har det jo egentlig blitt "håndtert" etter det ble laget en oppgave. 
Jeg tenker på håndtert som at et varsel har blitt sendt ut, og imens denne har ventet på å utgå så har det skjedd noe underveis i prosessen.
Så gjør det kanskje ikke noe at det blir plukket opp selv om det ikke er relevant? 🤔 Jeg tenkte det var clean å utelukkende hente varsler som er sendt ut, men som enda ikke har gått ut

Reply:
Der denne queryen brukes så sjekker man jo at svarfrist har gått ut på varselet (så det er mer enn tre uker siden varselet ble laget) og at det ikke har blitt publisert, i tillegg så sjekker man her at vurderingen.created_at > varsel.created_at. Usikker på om jeg skjønner helt hvorfor vi trenger denne ekstra sjekken da :thinking: Har du et testcase som feiler uten sjekken f.eks.?

Reply:
> 14 uker siden -> varsel lages
> 11 uker siden -> varsel går ut på dato -> oppgave lages -> veileder setter `UNNTAK`
> 8 uker siden -> ting endrer seg og veileder starter ny vurdering  -> sender ut forhåndsvarsel
> 5 uker siden -> varsel går ut på dato og vi sjekker på "håndterte" varsler -> dette aktivitetskravet vil da dukke opp som håndtert mens det egentlig ikke er håndtert.

Prøvde så godt jeg kunne å beskrive casen jeg tenker på over her👆😅
Hvis vi her har et helt nytt `aktivitetskrav` hver gang man "starter på nytt" så vil ikke dette være et problem, men om man derimot har et `aktivitetskrav` for hvert sykefraværstilfelle og markerer en "ny vurdering" ved å sette status til `NY` så vil dette være et problem.
Vi kan godt ta en kjapp huddle så kan du forklare meg hva du tenker 😊 Bare send DM

Reply:
Trooor jeg er med nå :sweat_smile: I så fall hadde det vært fint å basere seg på varsel.svarfrist i stedet for å "duplisere" 3-ukers-logikken i sqlen. Men det blir nok uansett ikke et problem med tilstandsreglene som ikke tillater flere forhåndsvarsler på samme aktivitetskrav https://github.com/navikt/isaktivitetskrav/pull/148/files#diff-84a6daf47f5bb5e8d8ce1fa0dc355b557509a57e3fe7f1583555474a3c354376R81

Reply:
Fjerner sjekk 😊